core/app/models/spree/product.rb

```diff
@@ -133,10 +133,7 @@ def find_or_build_master
 
     self.allowed_ransackable_associations = %w[stores variants_including_master master variants]
     self.allowed_ransackable_attributes = %w[name slug]
-
-    def self.ransackable_scopes(_auth_object = nil)
```

Comment:
Did you investigate that apparently useless `_auth_object`? Why it was there?

Reply:
Good question. When it was added, it referenced the example from the Ransack.
https://activerecord-hackery.github.io/ransack/going-further/other-notes/#authorization-allowlistingdenylisting
https://github.com/solidusio/solidus/pull/1904/commits/024091461a6945c2951705a7ddb50321ccd82d52.

The new implementation like the others, still allow parameters, but don't do anything with them.

If a user wants to use it, I guess they can override the whole method and still have access to `allowed_ransackable_scopes` if they wish